yb-voyager/cmd/templates/schema_optimization_report.template

```diff
@@ -515,7 +515,7 @@
                </p>
                <ul style="margin: 0.5rem 0 0 0; padding-left: 1.5rem; color: #666; font-size: 0.9rem;">
                   <li><strong>--skip-performance-optimizations true</strong> - Skip performance optimizations (e.g. redundant index removal)</li>
-                  <li><strong>--skip-recommendations true</strong> - Skip sharding recommendations from assessment</li>
+                  <li><strong>--skip-sharding-recommendations true</strong> - Skip sharding recommendations from assessment</li>
```

Comment:
this could be interpreted as "range vs hash" decision. Let's think a little more about the name. Perhaps "skip-colocation-recommendations"?